File: src/main/java/alz/genfive/webshop/model/Order.java
package alz.genfive.webshop.model;

import alz.genfive.webshop.enumeration.ShippingState;

import java.util.Date;
import java.util.Map;

public class Order {
    private Long id;
    private User user;
    private Map<Product, Integer> items;
    private PaymentMethod payment;
    private Boolean paid;
    private ShippingState shippingState;
    private Date date;

}

File: webshop_frontend_angular/src/app/entity/Order.ts
import { ShippingState } from "../enumeration/ShippingState";
import { User } from "./User";
import { PaymentMethod } from "./PaymentMethod";
import { Product } from "./Product";

export interface Order{
    id: number;
    user: User;
    product: Product;
    payment: PaymentMethod;
    paid: boolean;
    shippingState: ShippingState;
    date: Date;
}